coresimd/arm/cmsis.rs

```diff
@@ -0,0 +1,318 @@
+#![allow(non_snake_case)]
```

Comment:
Could you add a top-level comment here linking to the document / documents that specifies these intrinsics?